openpathsampling/exports/trajectories/trrtrajectorywriter.py

```diff
@@ -0,0 +1,26 @@
+from openpathsampling.integration_tools import HAS_MDTRAJ
+import numpy as np
+from .core import TrajectoryWriter
+
+class TRRTrajectoryWriter(TrajectoryWriter):
+    """Write a trajectory to a Gromacs TRR file.
+
+    This inludes velocities, so it can be used for restarts.
+    """
+    def __init__(self):
+        if not HAS_MDTRAJ:  # -no-cov-
+            raise ImportError("MDTraj is not available")
```

Comment:
I can guess your reason, but would like for it to be mentioned explicitly:
Any reason why you don't allow this writer to write out selections? (in case people want velocities, but not necessary all waters for analysis after the fact?)
(If this proposed functionality is in-scope, I would probably make it explicit by writing a separate class for it that emits significant warnings)

Reply:
Yeah, this is supposed to be a lossless format. You could easily create a different subclass to allow MDTraj selections, too (or, now that MDAnalysis is LGPL, you could use it!)

In general, I think I'm going to aim to make lossless writers separate (maybe we'll even add a flag to them?) because we can use these as the "database source of truth" to replace storing everything in a single SimStore/NetCDFPlus file. Lossy formats should not be used for that, but do have potential value as trajectory exporters for external analysis.